rahuldkjain/github-profile-readme-generator
# Add pelican to static site generator skills

## What type of PR is this? (check all applicable)


## Description

Just add [pelican](https://getpelican.com/) as a static generator website skill.
Also add the associated icon.

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

Ensure that `pelican` is displayed in the profile editor with its icon and that the link to the website `https://getpelican.com` is working.
I was unable to run the application so I did not perform any test on my side.

## Added to documentation?

- [ ] readme (there does not seems to be a need to add that change to the readme).
